docs/integrations/aiogram_dialog.rst

```diff
@@ -16,6 +16,7 @@ How to use
 
 .. code-block:: python
 
+    from dishka import FromDishka
```

Comment:
we try to encapsulate such import in intergration. Just in case they want customization